### PR TITLE
hmmer 3.1b2 (new formula)

### PR DESCRIPTION
**Pull request by ozcan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/hmmer.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Hmmer < Formula
+  desc "Build profile HMMs and scan against sequence databases"
+  homepage "http://hmmer.janelia.org"
+  # doi "10.1371/journal.pcbi.1002195"
```
Comment (fxcoudert) [inline Formula/hmmer.rb]: Please remove DOI and tag

File: Formula/hmmer.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  url "http://eddylab.org/software/hmmer3/3.1b2/hmmer-3.1b2.tar.gz"
+  sha256 "dd16edf4385c1df072c9e2f58c16ee1872d855a018a2ee6894205277017b5536"
+
+  head do
+    url "https://svn.janelia.org/eddylab/eddys/src/hmmer/trunk"
+    depends_on "autoconf" => :build
+  end
+
+  option "without-test", "Skip build-time tests (not recommended)"
```
Comment (fxcoudert) [inline Formula/hmmer.rb]: Remove option, never run tests (we only run them for security software)

File: Formula/hmmer.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  def install
+    system "autoconf" if build.head?
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "check" if build.with? "test"
+    system "make", "install"
+
+    # Installing libhmmer.a causes trouble for infernal.
+    # See https://github.com/Homebrew/homebrew-science/issues/1931
+    libexec.install lib/"libhmmer.a", include
```
Comment (fxcoudert) [inline Formula/hmmer.rb]: Let's install what upstream wants, i.e. run `make install` and that's it.

**Comment by ozcan** [comment]
Closing this pull request. It seems this formula has been merged on other, ironically more recent pull request at https://github.com/Homebrew/homebrew-core/pull/22357.

**Pull request closed without merging**